buddyboss/buddyboss-platform
# PROD-807 Fixed#2469: Minor styling issue while replying to forum discussions

### All Submissions:






### Changes proposed in this Pull Request:

update HTML & CSS 

Fixes #2469 .

### How to test the changes in this Pull Request:

1. Go to any forum discussion
2. add a reply
3. add the same text which has been added in prior reply very fast
4. See error > https://prnt.sc/14hsrik

### Proof Screenshots or Video

https://www.loom.com/share/b8900a52472749c18dbe8bdbfc938364

### Other information:
